libraries/standard/mqtt/utest/mqtt_config.h

```diff
@@ -1,29 +1,26 @@
 #ifndef CONFIG_H
```

Comment:
header guard should be changed to MQTT_CONFIG_H_

Reply:
Updated